### PR TITLE
New cheat - Guests ignore rain

### PR DESCRIPTION
**Pull request by jacon500**
Adds a new cheat in the Peeps menu that makes guests ignore rain when deciding to enter a queue line of any ride, covered or uncovered.

### CODE REVIEW

**Comment by danidoedel** [comment]
You should not touch any other language files than english_uk.txt here... other languages are maintained here: https://github.com/OpenRCT2/Localisation

File: data/language/english_uk.txt [hunk 1]
```diff
@@ -3904,6 +3904,7 @@ STR_5562    :WARNING!
 STR_5563    :This feature is currently unstable, take extra caution.
 STR_5564    :Insert Corrupt Element
 STR_5565    :{SMALLFONT}{BLACK}Inserts a corrupt map element at top of tile. This will hide any element above the corrupt element.
+STR_5566	:Guests ignore rain when entering queues
```
Comment (danidoedel) [inline data/language/english_uk.txt]: You should use spaces instead of a tab here ;)

**Comment by Gymnasiast** [comment]
I don't find this very useful. We already have cheats to change weather and freeze it.

**Comment by PFCKrutonium** [comment]
I think it makes sense @Gymnasiast - With this enabled, you would be able to still have rain, without the bankrupting side effects.

**Comment by Gymnasiast** [comment]
By the look of it, you haven't merged and/or rebased properly.

**Comment by janisozaur** [comment]
@jacon500 You want to fetch latest `origin`, rebase on top of `origin/develop`, then squash all the commits into one. I assume your `origin` points to upstream.

https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by Gymnasiast** [comment]
Rebasing shouldn't introduce new commits, so I'm afraid you didn't get it right. Easiest thing to do might be to create a new branch and recreate your changes there.

**Pull request closed without merging**